docs/modules/security/pages/encryption.adoc

```diff
@@ -91,6 +91,9 @@ You set the encryption algorithm, the salt, password and the iteration count to
 for generating the secret key. You also need to set the `enabled` attribute to `true`.
 Note that all members should have the same encryption configuration.
 
+NOTE: Use a strong encryption algorithm. Refer to the latest NIST guidelines for additional guidance.
```

Comment:
Do we want to add a link to the NIST site: https://www.nist.gov/privacy-framework/resource-repository/browse/guidelines-and-tools ?